### PR TITLE
add Minio client plugin

### PR DESCRIPTION
**Pull request by penpyt**
Add Minio Client, https://docs.minio.io/docs/minio-client-quickstart-guide

MinIO is an object storage server compatible with Amazon S3 cloud storage service.

[Build Status](https://travis-ci.com/penpyt/asdf-mc)

### CODE REVIEW

File: plugins/mc [hunk 1]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/penpyt/asdf-mc.git
```
Comment (imbsky) [inline plugins/mc]: Insert a final newline here

Comment (penpyt) [reply plugins/mc]: fix

**Comment by imbsky** [comment]
Thanks @penpyt for your contribution!

**Comment by imbsky** [comment]
Sorry for the late review.

**Pull request merged**